app/components/UI/EvmAccountSelectorList/EvmAccountSelectorList.tsx

```diff
@@ -483,22 +514,44 @@ const EvmAccountSelectorList = ({
       navigate,
       styles.titleText,
       accountTreeSections,
+      renderSectionHeader,
+      renderSectionFooter,
     ],
   );
 
+  const onContentSizeChanged = useCallback(() => {
+    // Handle auto scroll to account
+    if (!accounts.length || !isAutoScrollEnabled) return;
+    if (accountsLengthRef.current !== accounts.length) {
+      let selectedAccount: Account | undefined;
+
+      if (selectedAddresses?.length) {
+        const selectedAddress = selectedAddresses[0];
+        selectedAccount = accounts.find((acc) =>
+          areAddressesEqual(acc.address, selectedAddress),
+        );
+      }
+      // Fall back to the account with isSelected flag if no override or match found
```

Comment:
Cosmetic: (feel free to ignore):
```suggestion

      // Fall back to the account with isSelected flag if no override or match found
```